test_runner/regress/test_disk_usage_eviction.py

```diff
@@ -107,22 +134,18 @@ def eviction_env(request, neon_env_builder: NeonEnvBuilder, pg_bin: PgBin) -> It
 
     # allow because we are invoking this manually; we always warn on executing disk based eviction
     env.pageserver.allowed_errors.append(r".* running disk usage based eviction due to pressure.*")
-    env.pageserver.allowed_errors.append(
-        r".* Changing Active tenant to Broken state, reason: broken from test"
-    )
 
-    # break the difficult to use initial default tenant, later assert that it has not been evicted
-    broken_tenant_id, broken_timeline_id = (env.initial_tenant, env.initial_timeline)
-    assert broken_timeline_id is not None
-    pageserver_http.tenant_break(env.initial_tenant)
-    (broken_on_disk_before, _, _) = poor_mans_du(
-        env, timelines=[(broken_tenant_id, broken_timeline_id)]
-    )
-    env.pageserver.allowed_errors.append(
-        f".*extend_lru_candidates.*Tenant {broken_tenant_id} is not active. Current state: Broken"
-    )
-
-    timelines = []
+    # remove the initial tenant
```

Comment:
Should it be possible to not create it in the first place? Maybe an argument to init_start, can be done separately.

Reply:
It's not as straight-forward because `NeonEnv.tenant_id` is not `Optional[TenantId]`. Try to change it and see the plethora of mypy errors. And there are actually many tests that rely on an `initial_tenant` being there.

Here's a draft PR, will progress independently: https://github.com/neondatabase/neon/pull/3917